paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -50,6 +52,7 @@ import kotlin.coroutines.CoroutineContext
 internal class PaymentOptionsViewModel @Inject constructor(
     private val args: PaymentOptionContract.Args,
     private val linkProminenceFeatureProvider: LinkProminenceFeatureProvider,
+    private val linkAccountHolder: LinkAccountHolder,
```

Comment:
As we do on other activities and in flow controller, we need to keep the current link account state and update accordingly on logouts / verifications. The last updated value is passed back to `FlowController` to keep accounts in sync.